skywalking/profile/profile_context.py

```diff
@@ -34,6 +34,24 @@
 from skywalking.utils.time import current_milli_time
 
 
+THREAD_MODEL = 'thread'
+try:
+    from gevent import monkey
+    import greenlet
+    from gevent.exceptions import BlockingSwitchOutError
+
+    if monkey.is_module_patched('threading'):
+        if greenlet.__version__ < '2.0.0':
```

Comment:
This looks error prone. Can you use the `packaging` module to parse and compare the version?